data_adapter_oemof/build_datapackage.py

```diff
@@ -91,8 +91,34 @@ def __split_timeseries_into_years(parametrized_sequences):
 
         return split_dataframes
 
-    def get_foreign_keys(self):
-        pass
+    @staticmethod
+    def get_foreign_keys(
+        process_busses: list,
+        mapper: Mapper,
+    ) -> list:
+        """
+        Writes Foreign keys for one process.
+        Searches in adapter class for sequences fields
+        :rtype: list
+        :param process_busses:
+        :param adapter:
+        :return:
+        """
+        new_foreign_keys = []
+        for bus in process_busses:
+            new_foreign_keys.append(
+                {"fields": bus, "reference": {"fields": "name", "resource": "bus"}}
+            )
+
+        for field in dataclasses.fields(mapper.adapter):
+            if mapper.is_sequence(field.type):
```

Comment:
Unfortunately, this will not work, as this would add all possible candidates, as `is_sequence` checks only if parameter can be a Sequence (not if it's actually a sequence). As `is_sequence` check in mapper is called after trying to find a scalar value - this is fine. But using it here would add parameters which are defined as scalars, but could be sequences (by definition) as well...

Reply:
Improved in dac63e3ea22e820d80c2cb336161c7b3bd5753b4 even though second check might be enough. 

Failed in current state of Data where scalars have an additional `region` named `DE` and the `timeseries` don't. Should I create a new dataset where there is no scalar data for `DE`? Unfortunately I am unaware on how regions are exploded in `data_adapter`.